src/ConfigProvider.php

```diff
@@ -23,11 +24,41 @@ public function getDependencies() : array
                 LinkGenerator\UrlGeneratorInterface::class => LinkGenerator\ExpressiveUrlGenerator::class,
             ],
             'factories' => [
-                HalResponseFactory::class => HalResponseFactoryFactory::class,
-                LinkGenerator::class => LinkGeneratorFactory::class,
+                HalResponseFactory::class                   => HalResponseFactoryFactory::class,
+                LinkGenerator::class                        => LinkGeneratorFactory::class,
                 LinkGenerator\ExpressiveUrlGenerator::class => LinkGenerator\ExpressiveUrlGeneratorFactory::class,
-                Metadata\MetadataMap::class => Metadata\MetadataMapFactory::class,
-                ResourceGenerator::class => ResourceGeneratorFactory::class,
+                Metadata\MetadataMap::class                 => Metadata\MetadataMapFactory::class,
+                ResourceGenerator::class                    => ResourceGeneratorFactory::class,
+            ],
+            'invokables' => [
+                // TODO Would you prefer an InvokableFactory instead?
```

Comment:
No. Using the `invokables` configuration allows usage with other DIC implementations.

Reply:
Done.